src/app/users/login.ts

```diff
@@ -120,7 +120,16 @@ export const loginWithKratos = async ({
   }
 
   const network = BTC_NETWORK
-  return createToken({ uid: user.id, network, kratosUserId: kratosUserIdValid })
+
+  const account = await AccountsRepository().findByUserId(user.id)
+  if (account instanceof Error) return account
+
+  return createToken({
+    uid: user.id,
+    accountStatus: account.status.toUpperCase(),
```

Comment:
is `accountStatus` really necessary?

This is a mutable value in the backend, so including this value in the token will lead to stale token.

What is the logic for including this in the JWT, versus doing a call to the backend to know the up-to-date status?

Reply:
My thinking here: a token represents the authority of an account. The status of that account is part of that.

For clients apps enforcing the account status, they'll need to check that status every time they acquire a token.

On tokens being stale, the problem there is that we allow them to live beyond their need.

Reply:
I also have doubts about this - lets discuss the trade offs at standup.

Reply:
I removed this and relied on the auth endpoint instead.